client/tools/rhncfg/config_common/rpc_wrapper.py

```diff
@@ -106,7 +111,7 @@ def init_server(self, myuri):
         import urllib
         self._uri = myuri
         typ, uri = urllib.splittype(self._uri)
```

Comment:
will not work for Python3. we need something like:

typ, uri = urllib.parse.splittype(self._uri) 

self._host, self._handler = urllib.parse.splithost(uri)